python/chemiscope/structures/_ase.py

```diff
@@ -499,3 +499,84 @@ def _get_shape_params_atom(prefix, shape_kind, dictionary, atom_i):
         )
 
     return shape
+
+
+def ase_make_paths(bead_frames):
```

Comment:
Not a fan of this name, it's a bit too generic for my taste 😄. How about 

```suggestion
def ase_merge_pimd_frames(bead_frames):
```

Or something like it?